manager/integration/tests/common.py

```diff
@@ -3251,18 +3252,24 @@ def wait_for_all_instance_manager_running(client):
 
     for i in range(RETRY_COUNTS):
         instance_managers = client.list_instance_manager()
-        node_to_engine_manager_map, node_to_replica_manager_map = {}, {}
+        node_to_engine_manager_map = {}
+        node_to_replica_manager_map = {}
+        node_to_instance_manager_map = {}
         try:
             for im in instance_managers:
                 if im.managerType == "engine" and im.currentState == "running":
                     node_to_engine_manager_map[im.nodeID] = im
                 elif im.managerType == "replica" and \
                         im.currentState == "running":
                     node_to_replica_manager_map[im.nodeID] = im
+                elif im.managerType == "aio" and \
+                        im.currentState == "running":
+                    node_to_instance_manager_map[im.nodeID] = im
```

Comment:
Do we need to check `engine` and `replica` after introducing `aio`?

Reply:
IIRC we still need them for upgrade tests.

Reply:
Regard about the concern on the time of wait:
The flow is:
1. putting the instance_managers to `node_to_engine_manager_map`, `node_to_replica_manager_map` and `node_to_instance_manager_map` according the the manager type.
2. check if any of the `len(node_to_<type>_manager_map) == len(node)`.
3. Loop through each `node_to_<type>_manager_map` to check for desire state.

In case when all instance managers are `aio` type, `node_to_engine_manager_map` and `node_to_replica_manager_map` will contain an empty array, hence should not check for  `wait_for_instance_manager_desire_state`.

Reply:
Sounds good! Thank you.